usmqe_tests/api/conftest.py

```diff
@@ -36,3 +36,32 @@ def invalid_session_credentials(request):
     invalid_token = request.param
     auth = TendrlAuth(token=invalid_token, username=username)
     return auth
+
+
+@pytest.fixture
+def cluster_reuse(valid_session_credentials):
```

Comment:
I understand that there will be some fixture like `new_cluster` but how will be invoked tests if I want to call only one specific test when there is no cluster imported? In all tests are hardcoded `cluster_reuse` fixtures. How can switch them to `new_cluster` or something?